lib/ansible/utils/module_docs_fragments/netapp.py

```diff
@@ -38,7 +38,9 @@ class ModuleDocFragment(object):
   username:
       required: true
       description:
-      - Username
+      - Username : This can be a Cluster-scoped or SVM-scoped account, depending on whether a Cluster-level or SVM-level API is required.
```

Comment:
`Username: This can be a Cluster-scoped or SVM-scoped account, depending on whether a Cluster-level or SVM-level API is required. For more information, please read the documentation U(https://goo.gl/BRu78Z).`

Remove spaced between `Username` and `:`
One line
Use `U(...)` for URLs